packages/cli/oclif.manifest.json

```diff
@@ -453,6 +453,18 @@
           "name": "graphiql-port",
           "type": "option"
         },
+        "localhost-port": {
+          "dependsOn": [
+            "use-localhost"
+          ],
+          "description": "Port to use for localhost. Only applicable when --use-localhost is specified.",
```

Comment:
Should this just enable localhost and not depend on `--use-localhost`? Is there a reason to require both?

Reply:
I think `--localhost-port` implying `--use-localhost` is fine.